packages/components/grid/src/grid.scss

```diff
@@ -108,3 +108,7 @@ tr:not(:first-of-type) td {
 .selection-count {
   font-weight: normal;
 }
+
+sl-checkbox {
```

Comment:
Could you make the selector a little more specific? Perhaps add a `select-toggle` class and use that here. Otherwise things may break in the future when other checkboxes are added to the grid.

Reply:
Yes I will